plugins/org.eclipse.n4js/src/org/eclipse/n4js/xtext/workspace/ProjectBuildOrderInfo.java

```diff
@@ -43,15 +38,16 @@
 	/**
 	 * A provider for {@link ProjectBuildOrderInfo} instances.
 	 */
-	public static class Provider implements com.google.inject.Provider<IOrderInfo<ProjectConfigSnapshot>> {
-		/** Injector to be used for creating instances of {@link #ProjectBuildOrderInfo()} */
+	public static class Provider {
```

Comment:
Doesn't Guice rely on this class implementing its Provider interface?
If not, maybe use a more concrete method name than "get" here as well.